scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -623,7 +641,7 @@ class MicrobeEditor{
     // table. It will be used automatically.
     void enqueueAction(EditorAction@ action)
     {
-        if(action.cost != 0)
+        if(action.cost != 0 && !freeBuilding)
```

Comment:
I suppose this keeps freebuild mode working correctly?

Reply:
You are correct.